### ISSUE TITLE
FCNC hyperparameter debug

### ISSUE THREAD

**Issue by asalvadorsalas** [asalvadorsalas/IFAE_NNTools#10]
The use of 
4 hidden layers
196 nodes
40% dropout
2048 batch size
learning rate 0.00428095
returns a dobule peaked NN score
Check if its for different regions, decay or what
From different peaks maybe can be reopened

in single GPU every epoch takes < 10 s
